department-of-veterans-affairs/va.gov-cms
# VACMS-0: Updated email typo.

## Description

no issue was filed

this is a simple email address typo change `vacmssuport@va.gov`  > `vacmssupport@va.gov`

## QA steps

As an admin, view user account `/user/1317` that doesnt have any sections assigned and click on `Contact VACMS Support to request access` link. Verify that `TO` email is equal to `vacmssupport@va.gov` without typos
